### ISSUE TITLE
OnAddClick on OnShoppingClickListener not catching first click

### ISSUE THREAD

**Issue by andregdias** [JeasonWong/ElemeShoppingView#5]
When the user presses the button, it shows the minus and plus signs and the amount (1) but the OnAddClick event is not triggered. Only when the user presses the plus sign again.

I suggest this change (ShoppingView.java line 249):

```java
if (mState == STATE_NONE) {
  mNum++;
  startMoveAnim();
  if (mShoppingClickListener != null) {
    mShoppingClickListener.onAddClick(mNum);
  }
}
```

**Comment by starktonys** [comment]
your answer is correct .  thx

**Comment by paulobmarcos** [comment]
Hi @andregdias ! I am having the same problem! You are right! @JeasonWong, are you planning on a fix?

**Comment by JeasonWong** [comment]
I have fixed in 1.0.5. Please upgrade to `'me.wangyuwei:shoppoing:1.0.5'` . @andregdias @starktonys @paulobmarcos .  😘😘😘
